app/controllers/order_items_controller.rb

```diff
@@ -0,0 +1,20 @@
+class OrderItemsController < ApplicationController
+  before_action :find_order, only: [:index]
+
+  def index
+    @order_items = Kaminari.paginate_array(@order.order_items)
+                           .page(params[:page])
+                           .per(Settings.order_item.per_page)
```

Comment:
query đã trả về array rồi nên dùng trực tiếp thôi e nhé
```suggestion
    @order_items = @order.order_items.page(params[:page]).per(Settings.order_item.per_page)
```

Reply:
query đó trả về ActiveRecord::Associations nên em phải dùng method đó ép về mảng mới hiểu được method per chị ạ. Còn dùng trực tiếp sẽ bị lỗi ạ